packages/node/test/fixtures/18-nested-tsconfig/functions/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "main": "index.js",
   "devDependencies": {
-    "@types/node": "12.6.1"
+    "@types/node": "12.6.1",
+    "typescript": "3.7.7"
```

Comment:
This was necessary to get this fixture to use version `3.7.7`. Not sure how this was working before, exactly.

Reply:
Maybe the test runner is using `npm` and it's resolving the `typescript` packages installed by the parent directory?

Also note that this test in particular I noticed that the ES5 transpilation for string concatenation changed and I wasn't entirely sure why, but it prompted me to update the probe: https://github.com/vercel/vercel/pull/9692/files#diff-08299ba5c1fe93eaa22b12bbb372e3ddf2e181a524bec1d94811a8f9c1225a5bL10.